packages/core/cli/src/__template__/sandbox/tsconfig.json

```
@@ -1,7 +1,8 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "baseUrl": "."
+    "baseUrl": "."{%%SANDBOX_TSCONFIG_PATHS%%
+    }
```

```suggestion
    "baseUrl": "."{%%SANDBOX_TSCONFIG_PATHS%%}
```

The placeholder is broken due to the whitespaces.

---

This is just the template file; formatting will be wrong here it's just used to generate the actual file

---

Good catch, thanks!